packages/account/src/Sections/Security/Passwords/__tests__/deriv-password.spec.js

```diff
@@ -38,25 +39,37 @@ describe('<DerivPassword />', () => {
                 name: /deriv password/i,
             })
         ).toBeInTheDocument();
+        expect(
+            screen.getByText(/use the to log in to deriv\.com, deriv go, deriv trader, smarttrader, and deriv bot\./i)
+        ).toBeInTheDocument();
         // expect BrandDerivRed not to be in the document
         expect(screen.queryByText(/BrandDerivRed/i)).toBeInTheDocument();
         // expect button with text change password to be in the document
         expect(screen.getByRole('button', { name: /change password/i })).toBeInTheDocument();
         // expect button with text unlink from to not be in the document
         expect(screen.queryByText(/unlink from/i)).not.toBeInTheDocument();
-    });
 
-    it('displays the correct platform information for non-MF clients', () => {
-        render(<DerivPassword {...mock_props} landing_company_shortcode='svg' />);
         const popover_wrapper = screen.getAllByTestId('dt_popover_wrapper');
-        // expect popover to have length of 4
         expect(popover_wrapper).toHaveLength(4);
+    });
+
+    it('displays the correct platform information for non-MF clients & restricted countries', () => {
+        render(<DerivPassword {...mock_props} financial_restricted_countries />);
+
+        expect(screen.getByText(/use the to log in to deriv\.com, deriv trader and deriv go\./i));
+
+        const popover_wrapper = screen.getAllByTestId('dt_popover_wrapper');
+        // expect popover to have length of 2
+        expect(popover_wrapper).toHaveLength(2);
         // expect button with text change password to be in the document
         expect(screen.getByRole('button', { name: /change password/i })).toBeInTheDocument();
     });
 
     it('displays the correct platform information for MF clients', () => {
-        render(<DerivPassword {...mock_props} />);
+        render(<DerivPassword {...mock_props} is_eu_user />);
+
+        expect(screen.getByText(/use the to log in to deriv\.com and deriv trader\./i)).toBeInTheDocument();
```

Comment:
```suggestion
        expect(screen.getByText(/use this to log in to deriv\.com and deriv trader\./i)).toBeInTheDocument();
```

Reply:
got the query from testing playground which removed <strong>deriv password</strong>
I'll update this to platform names

Thanks for noticing 👍